### PR TITLE
Remove duplicated methods

### LINKED ISSUE #316: Remove duplicated methods after update of PowerSystemUtils
The following methods will be moved to PowerSystemUtils with version 1.5. If that one is released, remove those duplicated methods:
  - [x] `GridAndGeoUtils#buildSafeLineString`
  - [x] `GridAndGeoUtils#buildSafeLineStringBetweenCoords`
  - [x] `GridAndGeoUtils#buildSafeLineStringBetweenPoints`
  - [x] `GridAndGeoUtils#buildSafeCoord`
  - [x] `GridAndGeoUtils#buildSafePoint`
  - [x] `GridAndGeoUtils#getCoordinateDistances`
  - [x] `GridAndGeoUtils#totalLengthOfLineString`

as well as their equivalent tests!

### PR DESCRIPTION
**Pull request by ckittl**
Closes #316

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,15 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   deprecated `TarballUtils`
 -   updated Indriya to version 2.1.2 to include fixes for serialization
 -   Reworking the time series source (one source per time series, distinct mapping source, factory pattern)
+-   BREAKING: Moved methods `buildSafe{Coord,Point,LineString,LineStringBetweenCoords,LineStringBetweenPoints}`, `totalLengthOfLineString` from `GridAndGeoUtils` to `GeoUtils`
+-   BREAKING: Moved `CoordinateDistance` to _PowerSystemUtils_
```
Comment (johanneshiry) [inline CHANGELOG.md:32]:
```suggestion
-   BREAKING: Moved `CoordinateDistance` to [_PowerSystemUtils_](https://github.com/ie3-institute/PowerSystemUtils)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,15 +28,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   deprecated `TarballUtils`
 -   updated Indriya to version 2.1.2 to include fixes for serialization
 -   Reworking the time series source (one source per time series, distinct mapping source, factory pattern)
+-   BREAKING: Moved methods `buildSafe{Coord,Point,LineString,LineStringBetweenCoords,LineStringBetweenPoints}`, `totalLengthOfLineString` from `GridAndGeoUtils` to `GeoUtils`
```
Comment (johanneshiry) [inline CHANGELOG.md:31]: Is this one still valid? Or are these methods now also part of the PSU?

Comment (ckittl) [reply CHANGELOG.md:31]: Yes, this one is valid. They have been moved to `GeoUtils` in PSU.

Comment (johanneshiry) [inline CHANGELOG.md:31]:
```suggestion
-   BREAKING: Moved methods `buildSafe{Coord,Point,LineString,LineStringBetweenCoords,LineStringBetweenPoints}`, `totalLengthOfLineString` from `GridAndGeoUtils` to `GeoUtils` in [_PowerSystemUtils_](https://github.com/ie3-institute/PowerSystemUtils)
```

**Review by johanneshiry — CHANGES_REQUESTED** [review]

**Review by johanneshiry — APPROVED** [review]

**Pull request merged**